repository/src/main/java/org/alfresco/schedule/ScheduledJobLockExecuter.java

```diff
@@ -56,7 +59,7 @@ public class ScheduledJobLockExecuter
 {
     private static final long LOCK_TTL = 30000L;
 
-    private static Log logger = LogFactory.getLog(ScheduledJobLockExecuter.class.getName());
+    private static final Log logger = LogFactory.getLog(ScheduledJobLockExecuter.class);

```

Comment:
This constant should probably be upper case - `LOGGER`. We could also replace the logger using slf4j to remove the need for the defensive if statements.

Reply:
Changed logger to slf4j. Changed name to uppercase. Removed defensive if statements.